.github/workflows/ci.yml

```diff
@@ -56,6 +56,29 @@ jobs:
           A0_IASQL_API_TOKEN: ${{ secrets.A0_IASQL_API_TOKEN }}
           AWS_REGION: us-east-1
 
+  readonly-test:
+    runs-on: ubuntu-latest
+    name: readonly tests
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Run common tests
+        run: yarn coverage **/common/*
```

Comment:
this should be:
```suggestion
        run: yarn coverage **/readonly/*
```

Reply:
another option is that the read only test could live inside the `common` directory, we remove this new ci job and update the common one adding the read only credentials?